### DISCUSSION TITLE
I have a problem...

### DISCUSSION THREAD

**Discussion by rzsimpson01** [audacious-media-player/audacious#104] [Q&A]
I am currently running Audacious 4.1 on Linux Mint 21.3. Occasionally it works but usually it comes up with the playlists and transport but without the time slider? If I press play, or double click a song the "bubble pops" and I have to reopen the app. If I upgrade to 4.3.1 it won't run at all, it just pops up and then disappears.

Linux Mint Cinnamon 21.3
Toshiba Satellite C55-A laptop
8gb ram
1tb SSD
![Screenshot from 2024-01-20 14-52-33](https://github.com/audacious-media-player/audacious/assets/157246061/7ba94ce0-78d0-4608-9f0b-0938ff2a6440)

**Comment by radioactiveman** [comment]
Please explain better what you mean with "bubble pops".
And does this happen with every file or only some special ones? What is their file type?

Also post the output of `audacious -V`, preferably with the current version 4.3.1.
